### ISSUE TITLE
Roblox Crashes and Potato Mode Issues with Specific Accounts in Pet Simulator 99

### ISSUE THREAD

**Issue by ebann1** [ic3w0lf22/Roblox-Account-Manager#453]
Hi there,

I’m experiencing an issue with Roblox Account Manager after the most recent update. Specifically, when running Pet Simulator 99, Roblox windows for three of my eight accounts randomly crash, displaying a pop-up stating that Roblox needs to close. Here are the details:

Steps to Reproduce:

Run Pet Simulator 99 on Roblox Account Manager.
Monitor the accounts in use.
Expected Behavior:
The accounts should remain stable and not crash.

Actual Behavior:
Three of the eight accounts consistently crash with a pop-up that Roblox needs to close. This issue began occurring after the recent update of RAM.

Additional Information:

The problem is isolated to three specific accounts out of eight.
These three accounts are also the only ones showing as being in Potato Mode when checking the settings in Roblox Account Manager. Despite appearing to be in Potato Mode, they still display other players' pets.
Toggling the Potato Mode switch on and off clears out other players' pets, confirming that Potato Mode is indeed activated.
The other five accounts do not exhibit these issues.
Troubleshooting Steps Taken:

Removed and re-added the problematic accounts to see if the issue persists (awaiting results).
Impact:
The pop-up window causing Roblox to close disrupts the auto-clicker functionality, leading to timeouts and log-offs for the remaining accounts.

Request:
Could you please investigate this issue and provide guidance or a fix? It would be greatly appreciated.

Thank you for your assistance!

**Comment by ebann1** [comment]
I can confirm after removing and adding all of my accounts, three of them in fact still remaining in this false potato mode in pet simulator 99. Usually it takes about an hour or so for the first Roblox client to crash so I will update accordingly.

**Comment by ebann1** [comment]
Can confirm its still crashing.

**Comment by shadowbones222** [comment]
if you open task manager you will see that that the roblox ram usage for certain number of roblox windows become super high increasing from 1300mb to almost 4k mb over time, why does this happen, reason why the game fps become lowered and freezes as it makes the game crashes, hope there will be a fix?

**Comment by shadowbones222** [comment]
![Screenshot_2](https://github.com/user-attachments/assets/982ffb6a-4cf4-4570-b541-47ee526ea5ea)
as you can see the ram usage for the first window is increased to almost to 4gb uses while the rest is slowly going up i don't know why this happens

**Comment by shadowbones222** [comment]
just test some things and it is the game itself, not the roblox account manager

**Comment by ebann1** [comment]
Oh yeah I wish it was only 4 gigs....

<img width="386" alt="Screenshot 2024-07-23 050812" src="https://github.com/user-attachments/assets/04cc2d08-1e4c-4421-8e48-6aee5591859c">

This is definitely an issue but I'm still not quite sure this is what's making my app crash. I'm going to reproduce it and check logs.

**Comment by ebann1** [comment]
It definitely seems to make sense because the three accounts of mind that have potato mode on are also the ones that are crashing indicating that the extra graphics and memory are just too much. Right now I'm looking into memory managers or something that can trim memory. We're also even running everything in a VM which would be a pain in the butt but whatever.

**Comment by shadowbones222** [comment]
> Oh yeah I wish it was only 4 gigs....
> 
> <img alt="Screenshot 2024-07-23 050812" width="386" src="https://private-user-images.githubusercontent.com/41215701/351318661-04cc2d08-1e4c-4421-8e48-6aee5591859c.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjE3NDA1NDIsIm5iZiI6MTcyMTc0MDI0MiwicGF0aCI6Ii80MTIxNTcwMS8zNTEzMTg2NjEtMDRjYzJkMDgtMWU0Yy00NDIxLThlNDgtNmFlZTU1OTE4NTljLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA3MjMlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNzIzVDEzMTA0MlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWEyNDM5M2U3ZTAwZTMwZjFiZjA5MTUwMjdmOTFiYjIxOGQwOTJlZWNhMGRmZTM4MzUwYmQ3YzRmYmYwZWZlMTkmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.QPEFSB1qRpfRw7s5SQWmceu4ImUcZks7s6V5we0oyOE">
> This is definitely an issue but I'm still not quite sure this is what's making my app crash. I'm going to reproduce it and check logs.

20k mb ram thats literally 20gb ram using for 1 roblox window what the heck, your pc run out of ram which make it crash i think

**Comment by shadowbones222** [comment]
I just tried afk last area with 99 huge with 2 glittering charms each and teleported to void spawn which make it so the pets don't spit out the particles of gems that travels to your character, after an hour the ram memory seems to be stabled around 1,500mb ram memory, so i think its the glittering charms, but that sucks cause now i can't auto hatch best egg for huge.

**Comment by shadowbones222** [comment]
> Well, the account with 20 gigs of RAM did have about 100 glittering charms however the accounts that are crashing don't have any so it seems like for my issue initially reported it's unrelated or does not Point towards the glittering charms. Three accounts that keep crushing are actually mostly free to play and were running fine for weeks unattended until this latest update.

yeah before the update it didnt happen even with the glittering charms, icewolf pls fix

**Comment by ebann1** [comment]
Well I think it's separate so you may want to open your own issue ticket. The reason I'm saying is that my issue with Run Away Ram has disappeared but I'm still having problems with these three clients crashing. And yet again it's still the same three accounts that have potato mode falsely displayed as on. None of my other accounts seem to be bothered by this.

**Comment by ebann1** [comment]
Latest update, I shut down the accounts that we're having the issue. I gave up on them but it's a pretty boring week so none of those accounts are really making much money anyway. 

So the technical update is Roblox seems to have done a update today and after rebooting none of my characters appear to be falsely showing potato mode and they are all correctly in potato mode and I did not have to tag on it off and back on to shut off all the pets and graphics so that's a good sign. I'll close the ticket once I can confirm that nothing is crashing as I'll be doing a 24-hour test starting now

**Comment by ebann1** [comment]
No dice, three accounts crashed overnight and now my main account is doing the weird potato mode thing where I have to toggle it to actually enable potato mode. It'd be great if the developer chimed in with any thoughts.
